docs/architecture.md

```diff
@@ -82,6 +82,7 @@ Dev Home currently has the following tools:
 - Extensions Library
 - [Windows customization](../tools/Customization/DevHome.Customization/Customization.md)
 - Utilities
+- Repository Management
```

Comment:
Please also add the three new package references in the mermaid diagram above.

Reply:
I believe I got all the project references.  I might have had 3 before.  Now I have only 2.